### DISCUSSION TITLE
Support custom wagmi InjectedConnector name and logo

### DISCUSSION THREAD

**Discussion by riskers** [family/connectkit#238] [Q&A] [answered]
```ts
const trustWalletConnector = new InjectedConnector({
  chains,
  options: {
    name: 'GN',
    shimDisconnect: true,
    getProvider: () => (typeof window !== 'undefined' ? (window as any).trustwallet : undefined),
  },
});

const wagmiConfig = createConfig(
  getDefaultConfig({
    appName: 'demo',
    walletConnectProjectId: 'gn-walletconnect',
    autoConnect: true,
    connectors: [trustWalletConnector],
    publicClient,
    webSocketPublicClient,

    appDescription: 'Your App Description',
    appUrl: 'https://family.co', // your app's url
  }),
);

<WagmiConfig config={wagmiConfig}>
  <ConnectKitProvider debugMode>
    <Component {...pageProps} />
  </ConnectKitProvider>
</WagmiConfig>
```

But `connectkit` show this connector title `Browser Wallet`:

<img width="470" alt="image" src="https://github.com/family/connectkit/assets/5653652/ce9c5d9a-a757-4798-9ba9-f34d96b4449d">

How can I add a custom wallet connector name and logo?

**Comment by lochie** [comment, marked as answer]
There is a [PR open](https://github.com/family/connectkit/pull/242) that we're looking at merging soon that will fix this specific issue, and custom wallet name/logo branding is something [we're working towards](https://github.com/family/connectkit/pull/184). It's worth noting that the `getDefaultConfig` setup that ConnectKit suggests using already has the `InjectedConnector` setup built-in and should work fine with Trust.

**Reply by 3hrab** [reply]
The injected connecter when used with connectkit, seems to not be able to find Rabby wallet. Which is weird since all other wallet packages will even trigger rabby via the metamask connector. Connectkit blocks this, or cannot find rabby when used in place of metamask.

**Reply by lochie** [reply]
Have been looking into this, looks like Rabby tries to mimic MetaMask so we're having to change some things to make this work. Hoping to have this fixed soon.

**Reply by 3hrab** [reply]
Awesome! Looking forward to it 😝

**Reply by riskers** [reply]
default name is `Browser Wallet`

how to define?
